rllib/algorithms/algorithm.py

```diff
@@ -3264,6 +3553,15 @@ def _should_create_evaluation_env_runners(cls, eval_config: "AlgorithmConfig"):
             or eval_config.evaluation_interval
         )
 
+    @classmethod
```

Comment:
nit: Add a TODO comment saying that we need to flexibilize the different env/offline components and move away from the currently hardcoded: 1 EnvRunnerGroup, 1 eval EnvRunnerGroup, 1 OfflineData, 1 OfflineEvalRunnerGroup, etc..